### ISSUE TITLE
Adds label ''status: included-in-next-release" by default

### ISSUE THREAD

**Issue by benwinding** [fastlane/github-actions#33]
Hey guys,

It's weird this seems to the label `status: included-in-next-release` to my PR by default. How do I prevent that? _(it's not applicable to me)_

**Details:**
- The PR in question: https://github.com/benwinding/firecache/pull/2
- The Workflow file: https://github.com/benwinding/firecache/blob/07a28443abbb8b9d5a550872585547edcc7447d3/.github/workflows/deploy.yml#L38-L44 
- The Workflow Run: https://github.com/benwinding/firecache/runs/593964258

**The PR:**
https://github.com/benwinding/firecache/pull/2
![image](https://user-images.githubusercontent.com/11782590/79519875-4b982a00-8094-11ea-8859-dac4c018f582.png)

**The run**
https://github.com/benwinding/firecache/runs/593964258
Looks like this is where the defaults are added?
![image](https://user-images.githubusercontent.com/11782590/79519856-3de2a480-8094-11ea-905f-075c1e08c07a.png)

How can I stop that?

Thanks for open-sourcing your project, works really well :ok_hand: 

Cheers,
Ben

**Comment by benwinding** [comment]
Looks like this is the culprit.
https://github.com/fastlane/github-actions/blob/d0c340cd5e1c241ed6a1c9fbafbadce8577bc8d7/communicate-on-pull-request-released/action.yml#L11-L13
Should I just pass in nothing to this parameter?
Maybe `pr-label-to-add: ''`

**Comment by mollyIV** [comment]
Hello @benwinding 👋 

Unfortunately, there's no way to skip adding the label for now. If you set `pr-label-to-add: ''`, you'd get the error:

```
##[error]Validation Failed: {"value":"","resource":"Label","field":"name","code":"invalid"}
##[debug]Docker Action run completed with exit code 1
##[debug]Finishing: Communicate on PR merged
```

However, I think that making adding / removing labels optional for this action could be a great addition. The implementation should be fairly simple - we'd need to check if a label to add has any value:

```typescript
// main.ts 

export async function run() {
...

  const labelToAdd = core.getInput('pr-label-to-add');
    if (labelToAdd) {
      await addLabels(client, prNumber, [labelToAdd]);
  }

...
```

We could apply similar logic for a `pr-label-to-remove`.

Would you like to give the implementation a try? 😊

**Comment by benwinding** [comment]
Hi @mollyIV,

Thanks for getting back to me, yes that sounds like a reasonable approach. 

> Would you like to give the implementation a try?

How should I go about trying that implementation? Should I fork this repo and try it like that?

Cheers,
Ben

**Comment by janpio** [comment]
Yes, forking and then creating a PR to this repository would be the way to go.
